test/reply-error.test.js

```diff
@@ -393,3 +394,43 @@ test('should throw an error due to custom serializer can not handle the payload
     t.fail('should not be called')
   })
 })
+
+test('should serialize request and response', t => {
+  t.plan(8)
+  const lines = []
+  const stream = writeStream((data, enc, cb) => {
```

Comment:
For which case do you need `flush-write-stream` ?

Reply:
All of them?

Reply:
The package provides a `flush` callback but you don't use it. Why don't use ?

```js
class EchoStream extends stream.Writable {
  _write(chunk, enc, next) {
    lines.push(JSON.parse(data.toString()))
    next();
  }
}
```

Reply:
This would be much simpler and you don't need a package for this.
```js
  const lines = []
  const stream = {
    write (chunk) {
      lines.push(JSON.parse(chunk))
    }
  }

  const fastify = Fastify({logger: { level: 'info', stream }})
```